src/System.Security.Cryptography.Primitives/src/Resources/Strings.resx

```diff
@@ -79,6 +79,9 @@
   <data name="Cryptography_CryptoStream_FlushFinalBlockTwice" xml:space="preserve">
     <value>FlushFinalBlock() method was called twice on a CryptoStream. It can only be called once.</value>
   </data>
+  <data name="Cryptography_DefaultAlgorithm_NotSupported" xml:space="preserve">
+    <value>This platform does not allow the automatic selection of an algorithm.</value>
```

Comment:
Nit: I almost feel like we'll want an fwlink in here to direct developers to documentation showing the correct usage pattern.